site/content/docs/user/quick-start.md

```diff
@@ -26,8 +26,8 @@ shown [here](https://golang.org/doc/code.html#GOPATH) if you encounter the error
 
 Without installing go, kind can be built reproducibly with docker using `make build`.
 
-Stable binaries are also available on the [releases] page. Stable releases are
-generally recommended for CI usage in particular.
+Stable binaries are also available on the [releases](https://github.com/kubernetes-sigs/kind/releases) page.
```

Comment:
can we define this at the end of the file instead please?

Reply:
Could you clarify what you mean please- do you want me to move the whole paragraph to the end?

Reply:
Check how the other links are defined , the URL is at the end of the file

Reply:
@BenTheElder @aojea Thanks, now fixed. There's an in-line URL in the paragraph above (`[here](https://golang.org/doc/code.html#GOPATH)`) which confused me

Reply:
ah, we should move that one too, but in another PR. thanks!